dev_docs/cxg.md

```diff
@@ -119,46 +116,61 @@ This optional field contains a copy of the category color table, which MAY be us
+
+
+These Corpora metadata properties WILL exist in the CXG `__cxg_group_metadata__` field named `corpora`: 
+* `schema_version`
+* `title`
+* `X_normalization`
+* `X_approximate_distribution`
 
+These Corpora metadata properties MAY exist in the CXG `__cxg_group_metadata__` field named `corpora`:
+* `contributors`
+* `default_embedding`
+* `project_name`
+* `project_description`
+* `project_links`
+* `preprint_doi`
+* `publication_doi`
+
+For example:
 ```
 {
     "corpora": {
-        "version": {
-            "corpora_schema_version": "1.0.0",
-            "corpora_encoding_version": "0.1.0",
-        }
+        "schema_version": "2.0.0",
+        "title": "Analysis of 8 datasets of healthy and diseased mouse heart",
+        "X_normalization": "CPM",
+        "X_approximate_distribution": "normal",
+        "project_name": "Diseased Mouse Heart Study"
     }
 }
 ```
 
-The `corpora` metadata, if present, MUST contain the version information. Optionality of other values in this object will follow the specifications set forth in the relevant Corpora schema specification (ie, optional fields are optional, required are present, etc), with the following changes:
-* the contents of `corpora_encoding_version` MUST be identical to the `cxg_version`, as this field is defined as the current object encoding version, *NOT* the source data encoding version.
-* the entire encoding will be JSON, rather than a hybrid Python/JSON encoding, but will otherwise follow the data structure defined by the AnnData Corpora encoding.
-* the `<obs_column>_colors` will be omitted in favor of `cxg_category_colors`
-
 ### Other Corpora fields
 
-All other Corpora schema fields will be encoded into a CXG using the conventions defined in the [Corpora Schema AnnData Implementation](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema_h5ad_implementation.md). For example, fields in `AnnData.obs` will be encoded in the CXG `obs`array as defined [above](#obs-and-var).
+All other Corpora schema fields will be encoded into a CXG using the conventions defined in the [Corpora Schema 2.0.0](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/2.0.0/corpora_schema.md). For example, fields in `AnnData.obs` will be encoded in the CXG `obs`array as defined [above](#obs-and-var).
 
 ### Compatibility with CXG 0.1.0
 
-For backwards compatibility and continuity with CXG version 0.1.0, the following MUST be implemented.
+For CXG v0.1.0 files that were created from [Corpora Schema 1.1.0](https://github.com/chanzuckerberg/single-cell-curation/blob/2dc6a6f7ea342715d8a400a3cead29c691745df7/schema/1.1.0/corpora_schema.md) H5AD files, the following WILL apply:
```

Comment:
Is this line's rewrite accurate?  I was confused by the usage of "MUST".  Presumably the hosted cellxgene FE code doesn't care where the required field values are _taken from_ in an H5AD file, only that some value be provided.

Reply:
I don't think this is correct -- IIRC, the versions that were created with Corpora schema are CXG 0.2.0 (see version history below)

Reply:
I'm a bit confused as to what do with this section--do we need it at all? Seems out of place for a 0.2.0 spec.  I would assume that if we have CXG 0.1.0 files currently in use, the content in this section would be better off in a separate 0.1.0 spec file, and explorer  code would simply be considered compatible with both 0.1.0 and 0.2.0.   If 0.1.0 files do not exist (contrary to `Created in early 2020, and in active use`, below), we could presumably just delete this section.

Reply:
I think there are two useful things you could do, which would improve our code:
1. define a canonical algo to sniff the schema version out of _any_ CXG (ie, is it 2.0.0, 0.1.0, ????)
2. define what, if any, defaults should be used for critical metadata when reading old schema.  This may be a null set.
I other words, tighten up how various apps interpret CXG backwards compat.

Reply:
Leaving this section as now edited, since it's still relevant for the remaining `/e/` datasets that are in CXG 0.1 format. I added a note above regarding this. We can remove this section once all the CXG 0.1 files are wiped from existence (at least in the production env). I don't think we should worry heavily about backwards compatibility at this point.

Added more info about CXG & Corpora schema detection in the Versions section, below.